access/src/main/java/edu/unc/lib/dl/ui/controller/SearchActionController.java

```diff
@@ -90,7 +87,7 @@ public String list(@PathVariable("pid") String pid, Model model, HttpServletRequ
     @RequestMapping("/list")
     public String list(Model model, HttpServletRequest request) {
         SearchRequest searchRequest = generateSearchRequest(request);
-        searchRequest.setRootPid(collectionsPid.getPid());
+   //     searchRequest.setRootPid(collectionsPid.getPid());
```

Comment:
delete this line?

Reply:
Can actually specify the rootPid using RepositoryPaths here too

Reply:
Updated